apps/calendar/js/router.js

```diff
@@ -67,6 +68,11 @@ Router.prototype = {
     this.last = ctx;
   },
 
+  _currentState: function(ctx, next) {
+    this.current = ctx;
```

Comment:
I'm going to update this code later today and simply use the global `window.history.state` instead (since the app already uses the history API)